test/0.4.24/lidooracle.test.js

```diff
@@ -338,86 +648,96 @@ contract('LidoOracle', ([appManager, voting, user1, user2, user3, user4, nobody]
+          assertEvent(receipt, 'ReportableEpochIdUpdated', { expectedArgs: { epochId: 6 } })
           await assertReportableEpochs(6, 5)
         })
 
-        it('member removal removes their data', async () => {
-          for (let epoch = 1; epoch < 6; epoch++) await app.reportBeacon(epoch, 32, 1, { from: user1 }) // this should be removed
-          await assertReportableEpochs(1, 5)
-
-          for (let epoch = 1; epoch < 6; epoch++) await app.reportBeacon(epoch, 64, 2, { from: user2 }) // this should be intact
-          await assertReportableEpochs(1, 5)
+        it('member removal removes all data', async () => {
+          for (let epoch = 1; epoch < 6; epoch++) {
+            await app.reportBeacon(epoch, START_BALANCE + 32, 1, { from: user1 })
+            await app.reportBeacon(epoch, START_BALANCE + 64, 2, { from: user2 })
+          }
+          await assertReportableEpochs(5, 5)
 
           await app.removeOracleMember(user1, { from: voting })
           await assertReportableEpochs(5, 5)
 
-          await app.reportBeacon(5, 64, 2, { from: user3 })
+          await app.reportBeacon(5, START_BALANCE + 33, 2, { from: user2 }) // user2 reports again
+          await app.reportBeacon(5, START_BALANCE + 33, 2, { from: user3 })
           await assertReportableEpochs(5, 5)
 
-          await app.reportBeacon(5, 65, 2, { from: user4 })
+          await app.reportBeacon(5, START_BALANCE + 33, 2, { from: user4 })
           await assertReportableEpochs(6, 5)
         })
 
         it('tail member removal works', async () => {
-          for (let epoch = 1; epoch < 6; epoch++) await app.reportBeacon(epoch, 32, 1, { from: user1 }) // this should be intact
-          await assertReportableEpochs(1, 5)
-
-          for (let epoch = 1; epoch < 6; epoch++) await app.reportBeacon(epoch, 64, 2, { from: user4 }) // this should be removed
-          await assertReportableEpochs(1, 5)
+          for (let epoch = 1; epoch < 6; epoch++) {
+            await app.reportBeacon(epoch, START_BALANCE + 32, 1, { from: user1 })
+            await app.reportBeacon(epoch, START_BALANCE + 64, 2, { from: user4 })
+          }
+          await assertReportableEpochs(5, 5)
 
           await app.removeOracleMember(user4, { from: voting })
           await assertReportableEpochs(5, 5)
 
-          await app.reportBeacon(5, 32, 1, { from: user2 })
+          await app.reportBeacon(5, START_BALANCE + 32, 1, { from: user1 }) // user1 reports again
+          await app.reportBeacon(5, START_BALANCE + 32, 1, { from: user2 })
           await assertReportableEpochs(5, 5)
 
-          await app.reportBeacon(5, 32, 1, { from: user3 })
+          await app.reportBeacon(5, START_BALANCE + 32, 1, { from: user3 })
           await assertReportableEpochs(6, 5)
         })
 
         it('quorum change triggers finalization of last reported epoch, all other reportable epochs will be staled', async () => {
```

Comment:
Do we have a test for split voting with following configuration?

3 oracles vote for A
3 another oracles vote for B 
quorum is 3
and epoch shoudn't be finalized in that case

Reply:
won't it finalize at the third vote for A?